components/ui/PageHeader.vue

```diff
@@ -0,0 +1,47 @@
+<template>
+  <header class="page-header">
```

Comment:
Please, prefix this component with `The` to follow the [strong recommendation regarding single instance components](https://vuejs.org/v2/style-guide/#Single-instance-component-names-strongly-recommended).